maestro-cli/src/main/java/maestro/cli/command/TestCommand.kt

```diff
@@ -76,6 +76,9 @@ class TestCommand : Callable<Int> {
     @CommandLine.Parameters
     private lateinit var flowFile: File
 
+    @Option(names = ["--config"], description = ["Optional .yaml configuration file for Flows. If not provided, Maestro will look for a config.yaml file in the root directory."])
```

Comment:
```suggestion
    @Option(
      names = ["--config", "--workspace-config"],
      description = ["Optional YAML configuration file for Flows. If not provided, Maestro will look for a config.yaml file in the workspace's root directory."])
```

Let's make it clear it's workspace config